lib/movie_reserver.rb

```diff
@@ -1,7 +1,104 @@
 require 'csv'
+require 'pry'
+require_relative 'date_range'
+require_relative 'movie'
+require_relative 'rental'
+
 
 module GreenBox
   class MovieReserver
+    attr_accessor :movies,:rentals
+    # :reserver, :date_range
+
+    def initialize
+      @movies = MovieReserver.load_movies
+      @rentals = []
+      @date_range = GreenBox::DateRange.new(Time.parse('2018-08-08'),Time.parse('2018-08-09'))
+
+    end
+
+
+# `self.load_movies`: This method will open the csv file `movies.csv` and read in the movies and return an array of the given movies.  Note the actors are separated by the `:` character.  You will need to break up that field.
+    def self.load_movies
+      showtime_movies  = []
+
+      CSV.read('data/movies.csv', headers: false).each do |line|
+        movie_id = line[0].to_i
+        title = line[1]
+        publisher = line[2]
+        all_actors = line[3]
+        actors_names = all_actors.split(':')
+        actors = {actors: actors_names}
```

Comment:
You could also make this:

```ruby
actors = actors_names
```

And just use an array instead of a has containing one array.